### ISSUE TITLE
pretrain model

### ISSUE THREAD

**Issue by laihaoran** [ANTsRNet/BrainExtraction#8]
Hi, do you have  pretrain model for 3D brain extract?

**Comment by ntustison** [comment]
Yes.  If you use our [``brain_extraction``](https://github.com/ANTsX/ANTsPyNet/blob/master/antspynet/utilities/brain_extraction.py) program, it will automatically download the appropriate set of weights and build the model based on the requested modality.  [Here's](https://github.com/ANTsX/ANTsPyNet/blob/master/antspynet/utilities/get_pretrained_network.py#L33-L66) a list of all the pertained networks.
